mirrulations-validation/src/mirrval/job_validator.py

```diff
@@ -61,12 +60,10 @@ def check_for_missing_jobs(res):
 
 
 def generate_work(collection=None):
+    # commented out for static analysis reasons since
+    # the variable 'database' isn't being used.
+    # database = load_redis()
```

Comment:
I think the load_redis function needs to be added here.

Reply:
the 'database' variable isn't being used in that file. static analysis will throw errors so it is commented out for now. I didn't want to just outright delete it. when people in the validator are ready to use redis they can just comment out that line and it will load redis.